tools/docker/Dockerfile-inclavare-dev-alinux2

```diff
@@ -62,3 +62,6 @@ RUN yum install --nogpgcheck -y teesdk
 
 RUN echo "source /opt/alibaba/teesdk/intel/sgxsdk/environment" >> /root/.bashrc
 RUN ln -sf /usr/bin/cmake3 /usr/bin/cmake
+
+# install OPA go dependency
+RUN go get github.com/open-policy-agent/opa@v0.30.2
```

Comment:
Why `curl -L -o opa https://openpolicyagent.org/downloads/v0.30.2/opa_linux_amd64_static`. and `chmod 755 ./opa && mv opa /usr/local/bin/` is useless in alinux2?